### ISSUE TITLE
Mail by Hanne Vuegen 09/08/2022: AllergyIntolerance.code

### ISSUE THREAD

**Issue by bdc-ehealth** [hl7-be/allergy#70]
@HanneVuegen

-	Shouldn’t we specify CTI-extended > CNK > ATC
-	Shouldn’t we refer to SAMv2?

**Comment by bdc-ehealth** [comment]
CTI-extended, CNK and ATC are explicitly mentioned in the explanation next to the field. Do you want this to include the order of preference?

**Comment by HanneVuegen** [comment]
Yes and I believe it is useful information to add the link to SAM https://ramit.be/sam-v2/

**Comment by bdc-ehealth** [comment]
Added the reference to SAM v2
